docs/diagrams/DeleteTaskSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
```

Comment:
AddressBook has been changed to HeyMatez

Reply:
okay!